code/modules/projectiles/guns/medbeam.dm

```diff
@@ -124,3 +124,201 @@
 
 /obj/effect/ebeam/medical
 	name = "medical beam"
+
+//Damaged beamgun loot, DO NOT let it break
+#define SCREWDRIVER_OPEN 8
+#define REMOVE_OLD_PARTS 7
+#define WELD_SHELL 6
+#define INSTALL_LENS 5
+#define INSTALL_ELECTRONICS 4
+#define INSTALL_CELL 3
+#define MULTITOOL_ELECTRONICS 2
+#define SCREWDRIVER_CLOSED 1
+
+/obj/item/gun/medbeam/damaged
```

Comment:
I think this entire newly added item needs to be documented in the PR description